internal/text/text.go

```diff
@@ -108,11 +108,11 @@ func ParseColonSeparatedFailureFields(s string) ([]FailureField, error) {
 
 // Failure is a failure with a position in text.
 type Failure struct {
-	Filename string
-	Line     int
-	Column   int
-	ID       string
-	Message  string
+	Filename string `json:"filename,omitempty"`
+	Line     int    `json:"line,omitempty"`
+	Column   int    `json:"column,omitempty"`
+	LintID   string `json:"lintID,omitempty"`
```

Comment:
this should probably be `lint_id`, to be consistent with our json/yaml in the rest of Prottoool

Reply:
I went with camelCase here after I found a note on it from Google's style guide: https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format

I'm fine with using snake_case in this case though since it's more consistent with the rest of Prototool.